### ISSUE TITLE
When there are space between element tags, it is converted in replace function.

### ISSUE THREAD

**Issue by thorai219** [remarkablemark/html-react-parser#870]
## Expected Behavior

is it possible to not parse spaces between element tags?

## Actual Behavior

![Screenshot 2023-03-16 at 22 56 11](https://user-images.githubusercontent.com/23511973/225824853-71653fe7-26ed-4585-aac1-b95d9b4e9fb8.png)

## Reproducible Demo

here is a codesandbox: https://codesandbox.io/s/serene-babbage-ldo91s?file=/src/App.tsx

**Comment by remarkablemark** [comment]
I don't follow your question. Are you asking to exclude whitespace during parsing? If you don't want whitespace, can you run an HTML minifier?

**Comment by thorai219** [comment]
@remarkablemark yeah sorry, this has been resolved on my team. thank you for your time.
